src/ctia/import/capec.clj

```diff
@@ -0,0 +1,164 @@
+(ns ctia.import.capec
+  (:require [clojure.xml :as xml]
+            [ctim.schemas.ttp :as ttp]
+            [schema.core :as s]
+            [ctia.lib.time :as time]
+            [clojure.pprint :refer [pprint]]
+            [ctia.import.threatgrid.http :as http]))
+
+(defn capec-tag [kw]
+  (keyword (str "capec" kw)))
+
+(defn sieve
+  "Helps keep clojure.xml traversal code more concise, with sensible defaults."
+  ([e tag] (sieve e tag first :content))
```

Comment:
I don't mind creating your own solution but why not use zippers ?
http://blog.korny.info/2014/03/08/xml-for-fun-and-profit.html#parsing-xml-as-structured-data